src/state/apis/nft/selectors.ts

```diff
@@ -46,6 +48,7 @@ export const makeSelectNftItemsWithCollectionSelector = (accountIds: AccountId[]
     selectNftCollections,
     (nftUserTokens, collections): NftItemWithCollection[] => {
       const { data: nfts } = nftUserTokens
+
```

Comment:
When all is upserted, we don't need to consume the RTK query anymore, consume `portfolioNftAssetIds` and do a joiny-join

Reply:
https://github.com/shapeshift/web/commit/f2a88ceee3012202622ba5e314310e473bbbf218 and https://github.com/shapeshift/web/commit/8c01218bab15e6539524189a46519758c4b2cdff - keeping this opened for visibility